<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_gato</class>
 <widget class="QDialog" name="add_gato">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>50</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>ID: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>150</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Password:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="sp_id">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>50</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
<<<<<<< HEAD
   <property name="maximum">
    <number>999999</number>
   </property>
   <property name="displayIntegerBase">
    <number>10</number>
   </property>
=======
>>>>>>> b293d21c0dd70170a0f36cc02a3823017a8a924c
  </widget>
  <widget class="QDoubleSpinBox" name="sp_horas">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>180</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_password">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>150</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_usuario">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>120</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>220</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Aceptar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre de usuario:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_nombre">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>80</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Crear Gato</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>180</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Horas Trabajadas</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
